src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3097,6 +3097,16 @@ private bool AskUserToSaveWorkspacesOrCancel(bool allowCancel = true)
             return true;
         }
 
+        /// <summary>
+        /// Import Preferences
+        /// </summary>
+        /// <param name="path">Path from the xml settings file to be imported</param>
+        /// <returns>A PreferencesSettings instance from the selected path</returns>
+        public PreferenceSettings ImportPreferences(string path)
+        {
+            return model.ImportPreferences(path);
+        }
```

Comment:
Same here, wrong section as this is the private function region

Reply:
I couldn't find the specific place/region for the public methods like DynamoModel has, just moving it after the last public

Reply:
Do you consider useful group them by access level as a next action item for later ?